app.py

```diff
@@ -33,7 +33,7 @@ def create_response(data=None, message=None, status=200):
 # create a test route
 @app.route('/test', methods=['GET'])
 def test():
-  return make_response(jsonify({'message': 'test route'}), 200)
+    return create_response(message='Test route')
```

Comment:
this can be bad when, proper response code needs to be sent, and also, the intent was to send json objects as standard responses rather than just displaying some text.

So, would be better if these kind of changes are reverted back

Reply:
create_response helper sends a json object, not a text 🤔

Reply:
oh, my bad, i didnt notice the function `create_response`, i mistook it as make_responses.
everything seems good, now that indentation is also fixed. 
merging it